client/blocks/upwork-stats-nudge/index.js

```diff
@@ -0,0 +1,140 @@
+import { dismissNudge } from './actions';
+import { enhanceWithSiteType, recordTracksEvent } from 'state/analytics/actions';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { withEnhancers } from 'state/utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class UpworkStatsNudge extends Component {
+	static propTypes = {
+		isDismissed: PropTypes.bool.isRequired,
+		plan: PropTypes.object,
+		recordTracksEvent: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	componentDidMount() {
+		this.recordView();
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.siteId && this.props.siteId && this.props.siteId !== prevProps.siteId ) {
+			this.recordView();
+		}
+	}
+
+	recordView() {
+		if ( this.isVisible() ) {
+			this.props.recordTracksEvent( 'calypso_upwork_stats_nudge_view' );
+		}
+	}
+
+	onDismissClick = () => {
+		const plan = this.props.plan ? this.props.plan.productSlug : '';
+		this.props.recordTracksEvent( 'calypso_upworks_stats_nudge_dismiss_icon_click', {
+			plan,
+		} );
+		this.props.dismissNudge();
+	};
+
+	onStartNowClick = () => {
+		const plan = this.props.plan ? this.props.plan.productSlug : '';
+		this.props.recordTracksEvent( 'calypso_upwork_stats_nudge_start_now_button_click', {
+			plan,
+		} );
+	};
+
+	isVisible() {
+		return ! this.props.isDismissed;
+	}
+
+	render() {
+		if ( ! this.isVisible() ) {
+			return null;
+		}
+
+		return (
+			<Card className="upwork-stats-nudge">
+				<QueryPreferences />
+
+				<Gridicon
+					icon="cross"
+					className="upwork-stats-nudge__close-icon"
+					onClick={ this.onDismissClick }
+				/>
+
+				<SectionHeader
+					className="upwork-stats-nudge__header"
+					label={ this.props.translate( 'Recommendations from WordPress.com' ) }
+				/>
+
+				<div className="upwork-stats-nudge__body">
+					<div className="upwork-stats-nudge__image-wrapper">
+						<img
+							className="upwork-stats-nudge__image"
+							src="/calypso/images/upwork/illustration-builder-referral.svg"
+							alt={ this.props.translate( 'Your business with Google My Business' ) }
+						/>
+					</div>
+
+					<div className="upwork-stats-nudge__info">
+						<h1 className="upwork-stats-nudge__title">
+							{ this.props.translate( 'Need an expert to help realize your vision? Hire one!' ) }
+						</h1>
+						<p>
+							{ this.props.translate(
+								"We've partnered with Upwork, a network of freelancers with a huge pool of WordPress experts. They know their stuff and they're waiting to help you build your dream site."
+							) }
+						</p>
+						<div className="upwork-stats-nudge__button-row">
+							<Button
+								href={ '/experts/upwork?source=stat-banner' }
+								primary
+								onClick={ this.onStartNowClick }
+								target="_blank"
+								rel="noopener noreferrer"
+							>
+								{ this.props.translate( 'Find your expert' ) }
```

Comment:
Not mandatory at all but since we call `translate()` a lot in this method we could destructure it to save us from repeating code:

```javascript
render() {
  const { translate } = this.props;

  ...

  <SectionHeader
    className="upwork-stats-nudge__header"
    label={ translate( 'Recommendations from WordPress.com' ) }
  />
```

Reply:
I should go over to the Google my business and make this change!